doc/devel/coding_guide.rst

```diff
@@ -219,6 +219,17 @@ will run on all supported platforms and versions of Python.
 
   .. _tox: https://tox.readthedocs.io/
 
+* If you know your changes do not need to be tested (this is very rare!), all
+  CIs can be skipped by including ``[ci skip]`` or ``[skip ci]`` in your
+  commit message. If you know only a subset of CIs need to be run (e.g.,
+  if you are changing some block of plain reStructuredText and only want
+  only CircleCI to run to render the result), individual CIs can be skipped
```

Comment:
This is great! I've wanted this for a while. If I'm working on a PR, how do I turn this on and off? Just rewrite the commit message? Or does it only check the last commit?

Reply:
Only the latest commit

Reply:
Pushed a commit to clarify this, and included `[ci skip]` just to make sure it actually works